warehouse/bigquery/bigquery.go

```diff
@@ -826,15 +809,36 @@ func (bq *HandleT) LoadTable(tableName string) error {
 	return err
 }
 
-func (bq *HandleT) AddColumn(tableName, columnName, columnType string) (err error) {
-	err = bq.addColumn(tableName, columnName, columnType)
+func (bq *HandleT) AddColumns(tableName string, columnsInfo warehouseutils.ColumnsInfo) (err error) {
+	pkgLogger.Infof("BQ: Adding columns in table %s in bigquery dataset: %s in project: %s", tableName, bq.namespace, bq.projectID)
```

Comment:
I am wondering, are those logs useful? It seems to me that it is something frequent that might lead to noise.

Reply:
Whenever new properties are added, this will happen. Earlier we used to log for every insert statement. But now since we are batching for every 50 columns, we will see only a few logs.
Since this is just a one-time thing that will happen when new properties are added, I think it should be fine.